deploy/addons/registry/registry-rc.yaml

```diff
@@ -9,11 +9,11 @@ metadata:
 spec:
   replicas: 1
   selector:
-    addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/minikube-addons: registry
   template:
     metadata:
       labels:
-        addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
I think you need to keep this Reconcile label here in the template/metadata for the addon manager to pick it up.